app/css/components/footer-menu.css

```diff
@@ -0,0 +1,52 @@
+/* Footer menu */
+.footer-menu {
+  display: flex;
+
+  width: 100%;
+  height: 100%;
+}
+
+.footer-menu__item {
```

Comment:
As I said I don't really have opinions on style naming, so I leave this part entirely to you.
I'm happy as long as it's documented (which can wait on a subsequent pull request).